addons/danger/functions/fnc_checkBody.sqf

```diff
@@ -65,7 +65,7 @@ if !(_body isEqualTo []) exitWith {
                 _unit doFollow leader group _unit;
```

Comment:
when the Unit is alive and time is up but the distance is still bigger than 0.8 he continues to walk towards the target but does not do anything after that.
we maybe should move the doFollow always if the unit is alive

Reply:
Hmm. Not a bad idea. Rarely comes up as units which haven't reached the body yet have probably been given new orders. 

Perhaps rewrite a bit. With the waitUntilAndExecute syntax the timeout is a bit redundant in any case. https://cbateam.github.io/CBA_A3/docs/files/common/fnc_waitUntilAndExecute-sqf.html